### ISSUE TITLE
ProofView occasionally switches to very narrow width 

### ISSUE THREAD

**Issue by orilahav** [coq-community/vscoq#151]
At some seemingly random points the proof view starts using only around 10 chars per line, even though the tab is much wider. 

![Screen Shot 2020-07-30 at 11 30 31](https://user-images.githubusercontent.com/14070781/88900557-35cffe80-d258-11ea-9c7e-5fea40897ac1.png)

**Comment by TheoWinterhalter** [comment]
I also have the problem, and usually when it's there, it will come up very often unless I restart vscode.
A quick solution is to zoom in/zoom out as it renders again and often solves the problem, but going to a new goal or sometimes even just stepping will revert to the narrow display.

**Comment by mrhaandi** [comment]
This problem may be reproduced by first navigating to some proof and then set/unset the extension setting `Coq> Format: Enable`.

**Comment by maximedenes** [comment]
The right fix is probably to address https://github.com/coq-community/vscoq/issues/86. Working on it.

**Comment by Neilshweky** [comment]
Any updates on this? I'm having a similar issue.

**Comment by maximedenes** [comment]
I know this issue is very annoying (I suffer from it myself). I tried to fix it with the current code base, but failed. The way we parse XML today is very messy. My current plan is to finish #140 first, and output the proofview in a convenient format. We made good progress on https://github.com/coq/coq/pull/12461 (which is a prerequisite), but there is still a bit of work.

If someone wants to contribute a more ad-hoc fix meanwhile, PRs are welcome.

**Comment by Champitoad** [comment]
Hi,

I'm still having this issue, with the same symptoms as @TheoWinterhalter. I work on Windows with Coq 8.12.1 installed through opam on a WSL installation of Arch.

Also I tested with multiple projects, and it always seems to happen when this is the first time I open a proof script in VS Code. Once I have called the Coq: Quit or Coq: Reset command, it won't happen in subsequent goals and scripts.
